### ISSUE TITLE
Tooltips and PopupMenus can't be made transparent anymore (DisplayServer regression)

### ISSUE THREAD

**Issue by shmellyorc** [godotengine/godot#76071]
### Godot version

4.0

### System information

Windows 10

### Issue description

If you modify a theme with ToolTipPanel and you modify the texture of the tooltip, some odd color is behind it.

I tried going through almost every theme node and tried to set the color or panel empty and nothing removes the background color.

I even set the tooltip panel, panel to none/empty and it still shows some black texture behind it.

IMO, if its not broke, dont fix it

when ran:
![ApplicationFrameHost_H7004DMbK4](https://user-images.githubusercontent.com/5901214/232165919-29d9a898-c624-473e-a5cc-8c39a471a772.png)

texture:
![Aseprite_lhJJaFwi12](https://user-images.githubusercontent.com/5901214/232165955-b814b7d6-8046-4f81-9cc7-dadf2cb86edd.png)

theme menu thingy:
![Godot_Zhwfs9zFpf](https://user-images.githubusercontent.com/5901214/232165970-57526322-8a24-4943-9f7d-0289cbe4ac14.png)

### Steps to reproduce

1. Make a project
2. add a button
3. set the button tooltip text to anything
4. make a theme from the button
5. add tooltip panel to the theme
6. add panel style
7. set the panel style to styelboxtexture
8. add a texture
9. run the game
10. move your mouse over the button and wait for the tooltip to appear

it will show a color behind it. Very annoying.

### Minimal reproduction project

do what is shown in "Steps to reproduce"

**Comment by Calinou** [comment]
Is **Embed Subwindows** enabled in the advanced Project Settings? If not, tooltips are separate windows, which means that they cannot have things drawn behind unless per-pixel transparency is enabled for them. (There's currently no way to enable per-pixel transparency globally for all tooltips if they're separate windows.)

**Comment by folleah** [comment]
Столкнулся с такой же проблемой.

**Comment by YuriSizov** [comment]
@Zireael07 They just confirmed they have the same problem. But indeed, this is an international board and everyone should use English, even if through an automatic translator.

**Comment by aldocd4** [comment]
Hello, I had the same issue when using tooltip, still no solution on my side.

I'm overriding the TooltipPanel "panel" property to StyleboxEmpty for example, but it seems there is still a grey background on the tooltip.

![image](https://user-images.githubusercontent.com/2823944/236626146-2a18b7d0-f9b9-47cf-99c2-5c45fc569879.png)

Here is what happens when I set a red background color:

![image](https://user-images.githubusercontent.com/2823944/236626155-52fd41e6-75d0-4917-a10a-efcad07cdfc6.png)

Thanks.

**Comment by filkata** [comment]
Suffering from the same issue on 4.0.2, changing the **Embed Subwindows** setting had no effect.

**Comment by akien-mga** [comment]
#78823 has a minimum reproduction project: [tooltip_bug.zip](https://github.com/godotengine/godot/files/11903408/tooltip_bug.zip)

**Comment by Distantz** [comment]
Can confirm this issue on 4.1.1.

**Comment by kleonc** [comment]
The issue is the tooltip `PopupPanel` has no transparent background enabled (`TooltipPanel` extends `Viewport`).

As a workaround e.g. autoloading such script does the thing:
```gdscript
extends Node

func _enter_tree() -> void:
	get_tree().node_added.connect(on_node_added)

func on_node_added(node: Node) -> void:
	var pp := node as PopupPanel
	if pp and pp.theme_type_variation == "TooltipPanel":
		pp.transparent_bg = true
```

**Comment by alpapaydin** [comment]
same problem with 4.2.1

**Comment by AThousandShips** [comment]
Please open a new report as this was solved *in* 4.2 so you are experiencing a different bug, assuming you are configuring things correctly

**Comment by The2AndOnly** [comment]
Still a bug in 4.7, and I have tried every solution mentioned here and more
